File: sdk/java/driversdk/src/main/java/com/baidu/bce/sdk/context/models/Message.java
package com.baidu.bce.sdk.context.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Map;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Message {
    private String kind;
    private Map<String, String> meta;
    private Object content;
}

File: sdk/csharp/driversdk/DriverSDK/DmContext/Models/Message.cs
namespace DriverSDK.DmContext.Models
{
    public class Message
    {
        public string Kind { get; set; }
        public Dictionary<string, string> Meta { get; set; }
        public object Content { get; set; }

        public Message(string kind, Dictionary<string, string> meta, object content)
        {
            this.Kind = kind;
            this.Meta = meta;
            this.Content = content;
        }
    }
}

